src/style.css

```diff
@@ -0,0 +1,66 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir Heavy";
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-weight: 900;
+}
+
+* {
+  font-family: "Avenir", sans-serif;
+}
+
+body {
+  margin: 0;
+  --color-primary: rgba(61, 78, 97, 1);
+  --background-color: rgba(225, 231, 237, 1);
+  --shadow: 1px 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.search-container {
+  margin: 20px 8px 0;
+}
+
+.search-bar .input {
+  width: 100%;
+  box-sizing: border-box;
+  border: 1px solid var(--background-color);
+  border-radius: 4px;
+  background-image: url("./images/search.svg");
+  background-repeat: no-repeat;
+}
+
+.big-form .input {
+  margin-bottom: 20px;
+  background-size: 19px 19px;
+  height: 70px;
+  font-size: 16px;
+  padding-left: 62px;
+  background-position: 25px center;
+}
+
+.small-form .input {
```

Comment:
check this checklist point
<img width="967" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/102411130/0affa88f-9e01-4e56-962e-36f776ea42c7">